spec/features/stories_spec.rb

```diff
@@ -19,8 +19,8 @@
       visit project_path(project)
       wait_spinner
 
-      click_on 'Add story'
-
+      first('.column_header').click
```

Comment:
It waits until he can find one `.column_header` element and clicks on it. Since it is just a simple div without any click event attached. So, it would guarantee that the page is loaded.